### ISSUE TITLE
Doesn't warn cleanly about missing dependency for gtk-doc

### ISSUE THREAD

**Issue by hadess** [mesonbuild/meson#3738]
Using meson 0.45.1, I get this traceback instead of a nice warning telling me that gtk-doc isn't installed when generating docs in libfprint.

```
Building documentation for libfprint
Traceback (most recent call last):
  File "/usr/bin/meson", line 26, in <module>
    sys.exit(main())
  File "/usr/bin/meson", line 23, in main
    return mesonmain.run(sys.argv[1:], launcher)
  File "/usr/lib/python3.6/site-packages/mesonbuild/mesonmain.py", line 318, in run
    sys.exit(run_script_command(args[1:]))
  File "/usr/lib/python3.6/site-packages/mesonbuild/mesonmain.py", line 274, in run_script_command
    return cmdfunc(cmdargs)
  File "/usr/lib/python3.6/site-packages/mesonbuild/scripts/gtkdochelper.py", line 231, in run
    options.mode)
  File "/usr/lib/python3.6/site-packages/mesonbuild/scripts/gtkdochelper.py", line 113, in build_gtkdoc
    gtkdoc_run_check(scan_cmd, abs_out)
  File "/usr/lib/python3.6/site-packages/mesonbuild/scripts/gtkdochelper.py", line 55, in gtkdoc_run_check
    p, out = Popen_safe(cmd, cwd=cwd, env=env, stderr=subprocess.STDOUT)[0:2]
  File "/usr/lib/python3.6/site-packages/mesonbuild/mesonlib.py", line 721, in Popen_safe
    stderr=stderr, **kwargs)
  File "/usr/lib64/python3.6/subprocess.py", line 709, in __init__
    restore_signals, start_new_session)
  File "/usr/lib64/python3.6/subprocess.py", line 1344, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'gtkdoc-scan': 'gtkdoc-scan'
```

**Comment by nielsdg** [comment]
I think this might be solved already with commit 44ac680e5?

**Comment by dcbaker** [comment]
I think so too, so I'll close this. If it isn't fixed we can reopen
